README.md

```diff
@@ -1,4 +1,11 @@
-# Prometheus
+<h1 align="center" style="border-bottom: none">
+    <a href="//prometheus.io"><img alt="Prometheus" src="/web/ui/react-app/src/images/logo-prometheus-orange.png" width="80"></a><br>Prometheus
```

Comment:
I would just use the SVG version here so it scales automatically: https://upload.wikimedia.org/wikipedia/commons/3/38/Prometheus_software_logo.svg

For the logo file location, let's not put it into the React app (since it is not related to it), let's store it in https://github.com/prometheus/prometheus/tree/main/documentation/images instead.

Reply:
aa got it!